maintenance/wikia/sendWeeklyDigest.php

```diff
@@ -0,0 +1,74 @@
+<?php
```

Comment:
There are lots of files scattered in this directory. How about moving this to https://github.com/Wikia/app/tree/SOC-400/maintenance/wikia/cronjobs ?